docs-src/guides/add-new-blockchain.md

```diff
@@ -0,0 +1,67 @@
+# Add support for new blockchain
+
+This document contains a guide on how to add support for a new blockchain to the `caip10-link` doctype, as well as use for authentication.
+
+## Overview: Ceramic and blockchain accounts
+
+Ceramic interacts with blockchain accounts in two ways:
```

Comment:
There are actually 3 ways listed in this page: authentication, linking, and validation.  Why not list all 3 here and handle them equivalently?

Reply:
Validation is a counterpart of linking. Linking does not make sense without validation. Technically it is linking/validation (operations) that together comprise the linking process (human point of view). But, authentication is kind of independent. That was the reasoning.